provider/core.rb

```diff
@@ -332,9 +332,18 @@ def generate_objects(output_folder, types, version_name)
 
     def generate_object(object, output_folder, version_name)
       data = build_object_data(object, output_folder, version_name)
+      unless object.exclude_resource
```

Comment:
@rileykarson check out this new logic to exclude resource generation but still generate IAM policy. Will this work for TF?

Reply:
Sure!